internal/daemon/controller/tickers.go

```diff
@@ -155,3 +157,31 @@ func (c *Controller) startCloseExpiredPendingTokens(cancelCtx context.Context) {
 		}
 	}
 }
+
+func (c *Controller) startWorkerConnectionMaintenanceTicking(cancelCtx context.Context, m *cluster.DownstreamManager) {
+	const op = "controller.(downstreamManager).startManagingTicker"
+	timer := time.NewTimer(0)
```

Comment:
Can you add the getRandomInterval  that other tickers in this file use so that controllers aren't hitting the db at the same time?